mipengine/common/filters.py

```diff
@@ -0,0 +1,141 @@
+from mipengine.common.common_data_elements import common_data_elements
```

Comment:
I would suggest making all functions that are not used from outside private, not that it matters but to understand what functions are actually used from the "outside world".